index.html

```diff
@@ -12007,6 +12007,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>User agents MUST NOT change keyboard behavior in response to the <code>aria-keyshortcuts</code> attribute. Authors MUST handle scripted keyboard events to process <code>aria-keyshortcuts</code>. The <code>aria-keyshortcuts</code> attribute exposes the existence of these shortcuts so that assistive technologies can communicate this information to users.</p>
 				<p>Authors SHOULD provide a way to expose keyboard shortcuts so that all users can discover them, such as through the use of a tooltip. Authors MUST ensure that <code>aria-keyshortcuts</code> applied to disabled elements are unavailable.</p>
 				<p>Authors SHOULD avoid implementing shortcut keys that inhibit operating system, user agent, or assistive technology functionality. This requires the author to carefully consider both which keys to assign and the contexts and conditions in which the keys are available to the user. For guidance, see the keyboard shortcuts section of the <cite><a href="" class="practices"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite>.</p>
+				<p>If the page is translated authors SHOULD consider whether the keyboard shortcut is valid in the translated language or whether it needs to be changed.</p>
```

Comment:
```suggestion
				<p>If the page is translated authors SHOULD consider whether the keyboard shortcut will be valid in each translated language or whether it needs to be changed.</p>
```

Just a soft editorial suggestion here, to account for multiple localizations?